theme-kit/lib/createContentDeliveryClient.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import type { Category, Culture, Newsroom, PrezlyClient } from '@prezly/sdk';
+import { ApiError, NewsroomGallery, SortOrder, Stories, Story } from '@prezly/sdk';
+
+const DEFAULT_PAGE_SIZE = 20;
+
+interface Params {
+    formats?: Story.FormatVersion[];
+    pinning?: boolean;
+    pageSize?: number;
+}
+
+export function createContentDeliveryClient(
+    prezly: PrezlyClient,
+    newsroomUuid: Newsroom['uuid'],
+    {
+        formats = [Story.FormatVersion.SLATEJS_V4],
+        pinning = false,
+        pageSize: defaultPageSize = DEFAULT_PAGE_SIZE,
+    }: Params = {},
+) {
+    const contentDeliveryClient = {
```

Comment:
Wouldn't it be cleaner to use a class instead? IMO it's a bit harder to read in this form (especially when referencing the params)

Reply:
IDK, for some reason I don't like classes in JS. It feels to me a bit alien to the ecosystem :thinking: 

One particular advantage with this approach, is that there is no need to rely on `this` anywhere in this code. So you can destructure the object and use parts of it wherever you want.

Disregarding the cleanness (which is subjective), are there any performance differences between using both approaches that you are aware of?

Reply:
None that I'm aware of, this is purely subjective comment from me :)